docs/zephyr/general.rst

```diff
@@ -0,0 +1,22 @@
+.. _zephyr_general:
+
+General information about the Zephyr port
+=========================================
+
+The Zephyr Project is a Linux Foundation hosted Collaboration Project. It’s an open 
+source collaborative effort uniting developers and users in building a best-in-class 
+small, scalable, real-time operating system (RTOS) optimized for resource-constrained 
```

Comment:
try not to use advertising wording (like "best in class"), it should be informative only

(yes it's a little subjective what's informative or not!)